src/main/MQ2CommandAPI.cpp

```diff
@@ -962,6 +962,8 @@ void InitializeMQ2Commands()
 		{ "/removeaura",        RemoveAura,                 false, true  },
 #if HAS_ADVANCED_LOOT
 		{ "/advloot",           AdvLootCmd,                 true,  true  },
+		{ "/setlootfilter",     SetLootFilterCmd,           true,  true  },
+		{ "/removelootfilter",  RemoveLootFilterCmd,        true,  true  },
```

Comment:
I suggest that you add commands ('addfilter' and 'removefilter' maybe?) to the `/advloot` command instead of creating two new commands.